procedures/unit-testing-basics.ipf

```diff
@@ -3464,11 +3467,23 @@ Function RunTest(procWinList, [name, testCase, enableJU, enableTAP, enableRegExp
 		ClearBaseFilename()
 		CreateHistoryLog()
 
-		PathInfo home
-		if(!V_flag)
-			sprintf msg, "Error: Please Save experiment first."
-			UTF_PrintStatusMessage(msg)
-			return NaN
+		// Check if the current experiment needs to be saved. This is only required if certain features
+		// of UTF are used.
+		reqSave = 0
```

Comment:
I think you don't need the reqSave variable here. You can combine the conditions in the if.

Reply:
The problem here is that the if statement become very large and less understandable. It would help if I can wrap the code over multiple lines but this is not available in Igor Pro 6.

Reply:
Or maybe move the decision logif for reqSave into a function? This function can have multiple if/elseif statements with early return.

Reply:
During refactoring to get a separate function I noticed there is a better and shorter approach.